lib/taskana-core-test/src/test/java/acceptance/classification/query/QueryClassificationAccTest.java

```diff
@@ -0,0 +1,984 @@
+            .createClassificationQuery()
+            .validInDomainEquals(true)
+            .priorityIn(999)
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .list();
+
+    assertThat(classificationSummaryList).containsExactlyInAnyOrder(classificationSummary);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassification_When_QueryingForModifiedTimestamp() throws Exception {
+    String testIdentifier = UUID.randomUUID().toString();
+    Classification classification =
+        defaultTestClassification()
+            .customAttribute(CUSTOM_1, testIdentifier)
+            .buildAndStore(classificationService);
+    final Instant now = Instant.now();
+    Thread.sleep(10);
+    classificationService.updateClassification(classification);
+
+    List<ClassificationSummary> classificationSummaryList =
+        classificationService
+            .createClassificationQuery()
+            .customAttributeIn(CUSTOM_1, testIdentifier)
+            .modifiedWithin(new TimeInterval(now, null))
+            .list();
+
+    assertThat(classificationSummaryList).containsExactlyInAnyOrder(classification.asSummary());
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassification_When_QueryingForNameLike() throws Exception {
+    String testIdentifier = UUID.randomUUID().toString();
+    ClassificationSummary classificationSummary =
+        defaultTestClassification()
+            .name(testIdentifier + "Name")
+            .buildAndStoreAsSummary(classificationService);
+
+    List<ClassificationSummary> classificationSummaryList =
+        classificationService.createClassificationQuery().nameLike(testIdentifier + "%").list();
+
+    ClassificationSummary masterClassificationSummary =
+        classificationService.getClassification(classificationSummary.getKey(), "").asSummary();
+    assertThat(classificationSummaryList)
+        .containsExactlyInAnyOrder(classificationSummary, masterClassificationSummary);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassification_When_QueryingForNameIn() throws Exception {
+    String testIdentifier = UUID.randomUUID().toString();
+    ClassificationSummary classificationSummary =
+        defaultTestClassification()
+            .name(testIdentifier)
+            .buildAndStoreAsSummary(classificationService);
+
+    List<ClassificationSummary> classificationSummaryList =
+        classificationService.createClassificationQuery().nameIn(testIdentifier).list();
+
+    ClassificationSummary masterClassificationSummary =
+        classificationService.getClassification(classificationSummary.getKey(), "").asSummary();
+    assertThat(classificationSummaryList)
+        .containsExactlyInAnyOrder(classificationSummary, masterClassificationSummary);
+  }
+
+  @WithAccessId(user = "businessadmin")
+  @Test
+  void should_FindClassification_When_QueryingForDescriptionLike() throws Exception {
```

Comment:
There is no test that queries using descriptionIn. Can you add the test please?

Reply:
We don't have a descriptionIn query parameter. We should discuss this.